Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md

```diff
@@ -59,8 +81,14 @@ You cannot share indexes between servers and since Azure Web Apps use a shared f
 
 * Ensure you are using the [latest Examine version from Nuget](https://www.nuget.org/packages/Examine)
 * In ExamineIndex.config, you need to tokenize the path for each of your indexes to include the machine name, this will ensure that your indexes are stored in different locations for each machine. An example of a tokenized path is: `~/App_Data/TEMP/ExamineIndexes/{machinename}/Internal/`
-* In ExamineSettings.config, you can add this attribute to all of your indexers and searchers: `useTempStorage="Sync"`
-* The 'Sync' setting will store your indexes in ASP.Net's temporary file folder which is on the local file system. Lucene has issues when working from a remote file share so the files need to be read/accessed locally. Anytime the index is updated, this setting will ensure that both the locally created indexes and the normal indexes are written to. This will ensure that when the app is restarted or the local temp files are cleared out that the index files can be restored from the centrally stored index files. If you see issues with this syncing process (in your logs), you can change this value to be 'LocalOnly' which will only persist the index files to the local file system in ASP.Net temp files.
+* In ExamineSettings.config, you can add these properties to all of your indexers and searchers: 
+```
+useTempStorage="Sync"
+tempStorageDirectory="UmbracoExamine.LocalStorage.AzureLocalStorageDirectory, UmbracoExamine"
```

Comment:
We could sure but i guess the idea for docs is to try to minimize any reference to minimum version numbers. There is a plan to make docs much smarter when it comes to versioning but that hasn't been prioritized.